### PR TITLE
Beta 20161201 updates

### PR DESCRIPTION
**Pull request by jseldess**
Updated release notes for https://github.com/cockroachdb/cockroach/issues/11716

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/908)

### CODE REVIEW

**Comment by jseldess** [comment]
@sploiselle, would you please do a quick proof before I publish?

**Comment by sploiselle** [comment]
Review status: 0 of 14 files reviewed at latest revision, 8 unresolved discussions.

---

*[beta-20161110.md, line 21 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/908#-KXvsXyqcOD3euXWas8r:-KXvsXyqcOD3euXWas8s:b-6zv3wz) ([raw file](https://github.com/cockroachdb/docs/blob/6ab4b3f0e67956636ea9fd00c322e0d2be4c44ed/beta-20161110.md#L21)):*
> ```Markdown
> ### SQL Language Changes
> 
> - Adding a [`FOREIGN KEY`](constraints.html#foreign-keys) constraint now automatically creates any necessary index. [#9572](https://github.com/cockroachdb/cockroach/pull/9572)
> ```

nit: s/any/the?
or
s/any necessary index/necessary indexes

---

*[beta-20161201.md, line 38 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/908#-KXvsxd5uOINe7vOMJB8:-KXvsxd5uOINe7vOMJB9:b-nsth2i) ([raw file](https://github.com/cockroachdb/docs/blob/0233b61adbd9ab8cdf5fde82f8de6afb80a8af32/beta-20161201.md#L38)):*
> ```Markdown
> - The `ARRAY[]` constructor syntax is now supported. [#10585](https://github.com/cockroachdb/cockroach/pull/10585)
> - The `COLLATE` operator is now supported. Collation support is still incomplete and will be improved in upcoming releases. [#10605](https://github.com/cockroachdb/cockroach/pull/10605)
> - Keys in the `pg_catalog.pg_settings` table are now lowercase. [#10548](https://github.com/cockroachdb/cockroach/pull/10548)
> ```

"are now" sounds strange when discussing a previously unavailable feature

---

*[beta-20161201.md, line 39 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/908#-KXvt6Wv2BNIH4WoK3p7:-KXvt6Wv2BNIH4WoK3p8:b-nfehqv) ([raw file](https://github.com/cockroachdb/docs/blob/0233b61adbd9ab8cdf5fde82f8de6afb80a8af32/beta-20161201.md#L39)):*
> ```Markdown
> - The `COLLATE` operator is now supported. Collation support is still incomplete and will be improved in upcoming releases. [#10605](https://github.com/cockroachdb/cockroach/pull/10605)
> - Keys in the `pg_catalog.pg_settings` table are now lowercase. [#10548](https://github.com/cockroachdb/cockroach/pull/10548)
> - The `pg_catalog.pg_settings` now contains an entry for `max_index_keys`. [#10548](https://github.com/cockroachdb/cockroach/pull/10548)
> ```

same with "now"

---

*[beta-20161201.md, line 41 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/908#-KXvtBq8h87ojtEWLGFR:-KXvtBq8h87ojtEWLGFS:b-319hfj) ([raw file](https://github.com/cockroachdb/docs/blob/0233b61adbd9ab8cdf5fde82f8de6afb80a8af32/beta-20161201.md#L41)):*
> ```Markdown
> - The `pg_catalog.pg_settings` now contains an entry for `max_index_keys`. [#10548](https://github.com/cockroachdb/cockroach/pull/10548)
> - The `pg_catalog.pg_depend` table is now partially supported (just enough to support `pgjdbc`). [#10696](https://github.com/cockroachdb/cockroach/pull/10696)
> - The `current_schema` [function](functions-and-operators.html)n is now supported. [#10707](https://github.com/cockroachdb/cockroach/pull/10707)
> ```

Extra "n" on end of `[function](functions-and-operators.html)n`

---

*[beta-20161201.md, line 47 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/908#-KXvtQfcE8X9pJcDqBrW:-KXvtQfcE8X9pJcDqBrX:bs14jst) ([raw file](https://github.com/cockroachdb/docs/blob/0233b61adbd9ab8cdf5fde82f8de6afb80a8af32/beta-20161201.md#L47)):*
"one of four namespaces " -> would rephrase to focus on compatibility and remove ambiguity, e.g. 

>`pg_catalog` tables now report databases as belonging to PostgreSQL-compatible namespaces. This is for compatibility only; CockroachDB does not have a notion of namespaces.

---

*[beta-20161201.md, line 118 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/908#-KXvuxgOUoi99rd6vXOT:-KXvuxgOUoi99rd6vXOU:bgrm1w6) ([raw file](https://github.com/cockroachdb/docs/blob/0233b61adbd9ab8cdf5fde82f8de6afb80a8af32/beta-20161201.md#L118)):*
> ```Markdown
> ### Doc Updates
> 
> - Expanded the [cloud deployment](cloud-deployment.html) documentation to cover secure clusters: 
> ```

Include [Azure](deploy-cockroachdb-on-microsoft-azure.html)

---

*[beta-20161201.md, line 136 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/908#-KXvumM_K4SttcpyI8Gj:-KXvumMa7-xHoPcqG1uF:b-b0pviu) ([raw file](https://github.com/cockroachdb/docs/blob/0233b61adbd9ab8cdf5fde82f8de6afb80a8af32/beta-20161201.md#L136)):*
> ```Markdown
> - Updated [`SELECT`](select.html) statement documentation to cover using "index hints". [#894](https://github.com/cockroachdb/docs/pull/894) 
> - Updated [`FOREIGN KEY`](constraints.html#foreign-keys) constraint documentation to clarify that creating a table with a foreign key now automatically creates an index for you. [#895](https://github.com/cockroachdb/docs/pull/895)
> - Updated `RENAME` statement documentation to clarify that you can't rename a [database]((rename-database.html), [table](rename-table.html), [column](rename-column.html), or [index](rename-index.html) reference by a [view](views.html). [#873](https://github.com/cockroachdb/docs/pull/873)
> ```

Extra parentheses in `[database]((rename-database.html)` breaking link.

---

*[set-application-name.md, line 7 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/908#-KXvvErE-VPo7-MLCRNP:-KXvvErE-VPo7-MLCRNQ:btf12k4) ([raw file](https://github.com/cockroachdb/docs/blob/0233b61adbd9ab8cdf5fde82f8de6afb80a8af32/set-application-name.md#L7)):*
> ```Markdown
> ---
> 
> The `SET APPLICATION_NAME` [statement](sql-statements.html) sets the application name for the current session. This is a write-only variable that is used by some SQL frameworks, such as SQL Workbench/J, to improve query logging on the server. 
> ```

Seems worth noting the same thing that the release notes do. Something like, "Unless required by the framework, setting this variable produces no effect."

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/908)*

**Comment by jseldess** [comment]
Review status: 0 of 14 files reviewed at latest revision, 8 unresolved discussions.

---

*[beta-20161110.md, line 21 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/908#-KXvsXyqcOD3euXWas8r:-KXvw3JchSwSuNcPc6Y-:b-xf34j6) ([raw file](https://github.com/cockroachdb/docs/blob/6ab4b3f0e67956636ea9fd00c322e0d2be4c44ed/beta-20161110.md#L21)):*
<details><summary><i>Previously, sploiselle (Sean Loiselle) wrote…</i></summary><blockquote>

nit: s/any/the?
or
s/any necessary index/necessary indexes
</blockquote></details>

I can't quite figure out what I should change here. Seems like your comment applies to something else?

---

*[beta-20161201.md, line 38 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/908#-KXvsxd5uOINe7vOMJB8:-KXvwGMNaHR7cGkFots-:bys1tyw) ([raw file](https://github.com/cockroachdb/docs/blob/0233b61adbd9ab8cdf5fde82f8de6afb80a8af32/beta-20161201.md#L38)):*
<details><summary><i>Previously, sploiselle (Sean Loiselle) wrote…</i></summary><blockquote>

"are now" sounds strange when discussing a previously unavailable feature
</blockquote></details>

I think the keys were already there; they're just now lowercase.

---

*[beta-20161201.md, line 39 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/908#-KXvt6Wv2BNIH4WoK3p7:-KXvwiA1x_nj95lh-DWQ:b-icqbsq) ([raw file](https://github.com/cockroachdb/docs/blob/0233b61adbd9ab8cdf5fde82f8de6afb80a8af32/beta-20161201.md#L39)):*
<details><summary><i>Previously, sploiselle (Sean Loiselle) wrote…</i></summary><blockquote>

same with "now"
</blockquote></details>

This one reads ok to me, too. A topic for future debate ;)

---

*[beta-20161201.md, line 41 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/908#-KXvtBq8h87ojtEWLGFR:-KXvwwMaGxXeNz1hbL2B:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/0233b61adbd9ab8cdf5fde82f8de6afb80a8af32/beta-20161201.md#L41)):*
<details><summary><i>Previously, sploiselle (Sean Loiselle) wrote…</i></summary><blockquote>

Extra "n" on end of `[function](functions-and-operators.html)n`
</blockquote></details>

Done.

---

*[beta-20161201.md, line 47 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/908#-KXvtQfcE8X9pJcDqBrW:-KXvx9aWIF_lqCM87GCJ:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/0233b61adbd9ab8cdf5fde82f8de6afb80a8af32/beta-20161201.md#L47)):*
<details><summary><i>Previously, sploiselle (Sean Loiselle) wrote…</i></summary><blockquote>

"one of four namespaces " -> would rephrase to focus on compatibility and remove ambiguity, e.g. 

>`pg_catalog` tables now report databases as belonging to PostgreSQL-compatible namespaces. This is for compatibility only; CockroachDB does not have a notion of namespaces.
</blockquote></details>

Done.

---

*[beta-20161201.md, line 118 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/908#-KXvuxgOUoi99rd6vXOT:-KXvxiYY0JVkrZ5dbhf5:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/0233b61adbd9ab8cdf5fde82f8de6afb80a8af32/beta-20161201.md#L118)):*
<details><summary><i>Previously, sploiselle (Sean Loiselle) wrote…</i></summary><blockquote>

Include [Azure](deploy-cockroachdb-on-microsoft-azure.html)
</blockquote></details>

Done.

---

*[beta-20161201.md, line 136 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/908#-KXvumM_K4SttcpyI8Gj:-KXvxqKumPwWt6x39GzS:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/0233b61adbd9ab8cdf5fde82f8de6afb80a8af32/beta-20161201.md#L136)):*
<details><summary><i>Previously, sploiselle (Sean Loiselle) wrote…</i></summary><blockquote>

Extra parentheses in `[database]((rename-database.html)` breaking link.
</blockquote></details>

Done.

---

*[set-application-name.md, line 7 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/908#-KXvvErE-VPo7-MLCRNP:-KXvyVyyKvjwzWzYolFX:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/0233b61adbd9ab8cdf5fde82f8de6afb80a8af32/set-application-name.md#L7)):*
<details><summary><i>Previously, sploiselle (Sean Loiselle) wrote…</i></summary><blockquote>

Seems worth noting the same thing that the release notes do. Something like, "Unless required by the framework, setting this variable produces no effect."
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/908)*

**Comment by sploiselle** [comment]
Review status: 0 of 14 files reviewed at latest revision, 8 unresolved discussions.

---

*[beta-20161110.md, line 21 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/908#-KXvsXyqcOD3euXWas8r:-KXvz1Dai_cDMQ5nwkI8:b-yzdlii) ([raw file](https://github.com/cockroachdb/docs/blob/6ab4b3f0e67956636ea9fd00c322e0d2be4c44ed/beta-20161110.md#L21)):*
<details><summary><i>Previously, jseldess wrote…</i></summary><blockquote>

I can't quite figure out what I should change here. Seems like your comment applies to something else?
</blockquote></details>

"Adding a FOREIGN KEY constraint now automatically creates any necessary index." sounds strange to me. Suggest changing to:

"Adding a FOREIGN KEY constraint now automatically creates necessary indexes."

---

*[beta-20161201.md, line 38 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/908#-KXvsxd5uOINe7vOMJB8:-KXvzF6KSN688uHgDNQA:bc0ps3s) ([raw file](https://github.com/cockroachdb/docs/blob/0233b61adbd9ab8cdf5fde82f8de6afb80a8af32/beta-20161201.md#L38)):*
If the table is new, I don't think the keys were there before. Maybe this is my own misunderstanding of:

> The pg_catalog.pg_roles, pg_catalog.pg_description, pg_catalog.pg_settings, and pg_catalog.pg_index tables are now supported. 

That sounds to me like the tables (and their keys) weren't there before.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/908)*

**Comment by jseldess** [comment]
Review status: 0 of 14 files reviewed at latest revision, 2 unresolved discussions.

---

*[beta-20161110.md, line 21 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/908#-KXvsXyqcOD3euXWas8r:-KXw-auF_jrffe0aSNbN:bdodsrm) ([raw file](https://github.com/cockroachdb/docs/blob/6ab4b3f0e67956636ea9fd00c322e0d2be4c44ed/beta-20161110.md#L21)):*
<details><summary><i>Previously, sploiselle (Sean Loiselle) wrote…</i></summary><blockquote>

"Adding a FOREIGN KEY constraint now automatically creates any necessary index." sounds strange to me. Suggest changing to:

"Adding a FOREIGN KEY constraint now automatically creates necessary indexes."



</blockquote></details>

I see what you mean now. For some reason, reviewable applied this comment to the `beta-20161110.md` file.

---

*[beta-20161201.md, line 38 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/908#-KXvsxd5uOINe7vOMJB8:-KXw-mBTsNlx7MjCuNcN:b1hlaqf) ([raw file](https://github.com/cockroachdb/docs/blob/0233b61adbd9ab8cdf5fde82f8de6afb80a8af32/beta-20161201.md#L38)):*
<details><summary><i>Previously, sploiselle (Sean Loiselle) wrote…</i></summary><blockquote>

If the table is new, I don't think the keys were there before. Maybe this is my own misunderstanding of:

> The pg_catalog.pg_roles, pg_catalog.pg_description, pg_catalog.pg_settings, and pg_catalog.pg_index tables are now supported. 

That sounds to me like the tables (and their keys) weren't there before.
</blockquote></details>

Ah, I see. OK, I think those other two are redundant, so I'm removing them and grouping all `pg_catalog` together.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/908)*

**Comment by bdarnell** [comment]
LGTM

---

Review status: 0 of 14 files reviewed at latest revision, 3 unresolved discussions.

---

*[beta-20161201.md, line 38 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/908#-KXvsxd5uOINe7vOMJB8:-KXxjgldkFf42Z6XCaU7:b-r09haw) ([raw file](https://github.com/cockroachdb/docs/blob/0233b61adbd9ab8cdf5fde82f8de6afb80a8af32/beta-20161201.md#L38)):*
<details><summary><i>Previously, jseldess wrote…</i></summary><blockquote>

Ah, I see. OK, I think those other two are redundant, so I'm removing them and grouping all `pg_catalog` together.
</blockquote></details>

Yeah, I ended up with a few redundancies when I merged the 1110 and 1117 notes into the new file. 

---

*[beta-20161201.md, line 142 at r2](https://reviewable.io:443/reviews/cockroachdb/docs/908#-KXxk3qjSPAR0XbWPxEz:-KXxk3qjSPAR0XbWPxF-:b-w211qb) ([raw file](https://github.com/cockroachdb/docs/blob/a9d7bbf878df9160068d6bcbcf79fa6e9980d1ca/beta-20161201.md#L142)):*
> ```Markdown
> ### Contributors
> 
> This release includes 292 merged PRs by 30 authors. We would like to thank the following contributors from the CockroachDB community, including first-time contributors Christian Gati, Dustin Hiatt, kiran, and Nathan Johnson.
> ```

Doesn't really matter but the count is now 297 PRs with the last 5 updates. 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/908#-:-KXxkUokfzHvvIpnthJt:bb6ywmx)*
